NomadicNibbler/foodtruck-fe
# Fix/invalid address

## What does this PR do?

- Background image was selected and extras were removed
- When a user has an invalid address saved, they are now redirected upon login to change their address

## How should it be tested?

- create a new user and enter an invalid address and zip code such as 45678
- you should be redirected to enter a new location
- when you change your location to something valid, it will show you the map page with correct address
- when you log out and log back in you will be taken to the valid address that was saved

## What do the future iterations hold?

- refactoring error handling to make it a smoother user experience

## Linked Issues
